index.html

```diff
@@ -506,6 +547,7 @@ <h5 class="name">Sanmugathasan Prasanthan</h5>
     <p class="small text-muted text-left mb-0"> *Our mentors and speakers engage in mentoring and speaking voluntarily
         in their leisure time<br> and we don't have a direct affiliation with the mentioned organisations</p>
 </script>
+<script src="assets/js/back2top.js"></script>
```

Comment:
Shall we put the js script here without an external file?

Reply:
Having an external file would help us to include it on other pages as well.

Reply:
In that case, css should be also moved to a separate file.